src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -49,7 +50,7 @@ public class EditCommand extends Command {
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_DUPLICATE_PERSON = "There are no changes to make.";
```

Comment:
There is a chance that the edited person and the target are different but the edited person matches someone else. So I don't think should change?